patches/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -1,7 +1,10 @@
 --- a/net/minecraft/world/item/ItemStack.java
 +++ b/net/minecraft/world/item/ItemStack.java
-@@ -77,7 +_,7 @@
+@@ -75,9 +_,10 @@
+ import net.minecraft.world.level.block.Block;
+ import net.minecraft.world.level.block.state.BlockState;
  import net.minecraft.world.level.block.state.pattern.BlockInWorld;
++import net.neoforged.neoforge.event.entity.player.BlockInteractEvent;
```

Comment:
No import changes in patches

Reply:
Are we still not doing that?  I thought the recent DiffPatch allowed for this to be a non-issue

Reply:
We still aren't making import changes to patches because if too many import changes are made by mojang, the order and offsets will be screwed to the point where the misalignments will be frequent. (also, our diffpatch configuration is a bit *too* fuzzy, but that's another matter)

Reply:
Intellij tried to be helpful and I didn't notice, will try to fix